### ISSUE TITLE
Rack crashes when instantiating module browser

### ISSUE THREAD

**Issue by cschol** [gibbonjoyeux/VCV-Biset#23]
Backtrace:

```
[4.666 fatal adapters/standalone.cpp:49 fatalSignalHandler] Fatal signal 11. Stack trace:
26: ./Rack() [0x4043ed]
25: /lib/x86_64-linux-gnu/libc.so.6(+0x42520)
24: /home/cschol/src/Rack-2.0-Home/plugins-lin-x64/Biset/plugin.so(BlankWidget::set_panel(int)+0xb5)
23: /home/cschol/src/Rack-2.0-Home/plugins-lin-x64/Biset/plugin.so(BlankWidget::BlankWidget(Blank*)+0x10a)
22: /home/cschol/src/Rack-2.0-Home/plugins-lin-x64/Biset/plugin.so(rack::createModel<Blank, BlankWidget>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)::TModel::createModuleWidget(rack::engine::Module*)+0x4f)
21: ./libRack.so(rack::app::browser::ModelBox::draw(rack::widget::Widget::DrawArgs const&)+0x4a4)
20: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
19: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
18: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
17: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
16: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
15: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
14: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
13: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
12: ./libRack.so(rack::ui::ScrollWidget::draw(rack::widget::Widget::DrawArgs const&)+0x34)
11: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
10: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
9: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
8: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
7: ./libRack.so(rack::widget::Widget::drawChild(rack::widget::Widget*, rack::widget::Widget::DrawArgs const&, int)+0x124)
6: ./libRack.so(rack::widget::Widget::draw(rack::widget::Widget::DrawArgs const&)+0xcc)
5: ./libRack.so(rack::window::Window::step()+0x74c)
4: ./libRack.so(rack::window::Window::run()+0x28)
3: ./Rack(main+0xec2)
2: /lib/x86_64-linux-gnu/libc.so.6(+0x29d90)
1: /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80)
0: ./Rack() [0x404319]
```

**Comment by gibbonjoyeux** [comment]
Sorry about that !
Fixed hash: 8357c68f14d155e6935f954648ec80b2f61f8619

**Comment by cschol** [comment]
Confirmed fix.
